cpp/include/cuml/cuml.hpp

```diff
@@ -98,11 +98,20 @@ class cumlHandle {
   int getNumInternalStreams();
 
   /**
-   * @brief API to get the internal streams as a vector.
-   * @return vector of internal streams in the handle
+   * @brief API to get the worker streams as a vector.
+   * @return vector of worker streams in the handle
    */
   std::vector<cudaStream_t> getInternalStreams() const;
 
+  /**
+   * @brief API to get a specific worker stream by index.
+   * @return worker stream corresponding to index in the handle
+   */
+  std::vector<cudaStream_t> getInternalStream(int sid) const;
```

Comment:
This doesn't seem to have corresponding definition?

Reply:
Should also return `cudaStream_t` not a vector of them.